app/src/test/java/org/simple/clinic/home/HomeScreenControllerTest.kt

```diff
@@ -84,7 +75,23 @@ class HomeScreenControllerTest {
 
   @Test
   fun `when facility change button is clicked facility selection screen should open`() {
+    setupController()
     uiEvents.onNext(HomeFacilitySelectionClicked())
     verify(screen).openFacilitySelection()
   }
+
+  private fun setupController() {
+    clock.setDate(LocalDate.parse("2018-01-01"))
```

Comment:
Not necessary to change this, but we now can set the date of the clock in the constructor of `TestUserClock`.